topics/icrc_25_signer_interaction_standard.md

```diff
@@ -13,6 +13,13 @@ This specification describes a communication protocol between dapps (decentraliz
 * signer: A service that manages a user's keys and can sign and perform canister calls on their behalf.
 * relying party: A service that wants to request calls on a specific canister.
 
+## Types
+
+- `text`: A plain `string` value.
+- `blob`: A `string` value describing binary data encoded in base64.
+- `nat`: A `string` value of an unsigned 64-bit integer.
```

Comment:
nit: `nat` should probably be unbounded (as in Candid)

Reply:
Thanks. Although it will not be addressed here, since it is handled in #78.